docs/guides/high-availability/concepts-memory-and-cpu-sizing.adoc

```diff
@@ -29,83 +29,84 @@ Summary:
 
 Recommendations:
 
-* The base memory usage for an inactive Pod is 1000 MB of RAM.
-
-* For each 100,000 active user sessions, add 500 MB per Pod in a three-node cluster (tested with up to 200,000 sessions).
-+
-This assumes that each user connects to only one client.
-Memory requirements increase with the number of client sessions per user session (not tested yet).
+* The base memory usage for a Pod including caches of Realm data and 10,000 cached sessions is 1250 MB of RAM.
 
 * In containers, Keycloak allocates 70% of the memory limit for heap based memory. It will also use approximately 300 MB of non-heap-based memory.
 To calculate the requested memory, use the calculation above. As memory limit, subtract the non-heap memory from the value above and divide the result by 0.7.
 
-* For each 45 password-based user logins per second, 1 vCPU per Pod in a three-node cluster (tested with up to 300 per second).
+* For each 15 password-based user logins per second, allocate 1 vCPU to the cluster (tested with up to 300 per second).
 +
 {project_name} spends most of the CPU time hashing the password provided by the user, and it is proportional to the number of hash iterations.
 
-* For each 500 client credential grants per second, 1 vCPU per Pod in a three node cluster (tested with up to 2000 per second).
+* For each 200 client credential grants per second, 1 vCPU to the cluster (tested with up to 2000 per second).
 +
 Most CPU time goes into creating new TLS connections, as each client runs only a single request.
 
-* For each 350 refresh token requests per second, 1 vCPU per Pod in a three-node cluster (tested with up to 435 refresh token requests per second).
+* For each 120 refresh token requests per second, 1 vCPU to the cluster (tested with up to 435 refresh token requests per second).
 
 * Leave 200% extra head-room for CPU usage to handle spikes in the load.
 This ensures a fast startup of the node, and sufficient capacity to handle failover tasks like, for example, re-balancing Infinispan caches, when one node fails.
 Performance of {project_name} dropped significantly when its Pods were throttled in our tests.
 
+* By default {project_name} will store user sessions in the database. For the database, allocate per 100 requests/second that update the database (login, logout, refresh token) 0.35-0.7 vCPU and 1400 WriteIOPS on an Aurora PostgreSQL multi-AZ database base on a 2 vCPU `db.t4g.large` instance type (tested up to 400 requests per second).
```

Comment:
Would you consider to re-phrase the two statements here for better readability?

```suggestion
{project_name}, which by default stores user sessions in the database, requires the following resources for optimal performance on an Aurora PostgreSQL multi-AZ database:

For every 100 login/logout/refresh requests per second:
- Allocate between 0.35 and 0.7 vCPU.
- Ensure 1400 Write IOPS are available.

Instance type recommendations:
- db.t4g.large (2 vCPU) is suitable for typical loads.
- db.t4g.xlarge (4 vCPU) is recommended for peak usage to avoid slowdowns.

Key takeaway:  Higher CPU usage on the database can lead to slower response times, particularly during peak loads. Choose a larger instance type if faster response times are critical.
```

Reply:
Thank you for this suggestion. I suppose the "Instance type recommendations" will need some more context as the understanding of "typical loads" will vary. I'll add that context in another commit.